src/qibolab/dummy.py

```diff
@@ -232,6 +323,25 @@ def create_dummy():
         qubit.flux = channels[f"flux-{q}"]
         qubit.twpa = channels["twpa"]
 
+    # map channels to couplers
+    for c, coupler in couplers.items():
+        coupler.flux = channels[f"flux_coupler-{c}"]
+
+    # FIXME: Call couplers by its name
+    # assign couplers to qubits
+    for c in itertools.chain(range(0, 2), range(3, 5)):
+        qubits[c].flux_coupler[c] = couplers[c].name
+        qubits[2].flux_coupler[c] = couplers[c].name
+
+    # FIXME: Call couplers by its name
+    # assign qubits to couplers
+    for c in itertools.chain(range(0, 2), range(3, 5)):
```

Comment:
This `itertools.chain(...)` is the same repeated three times: I'd just give it a name, and use the variable (it would make simpler to understand what's happening).

Reply:
Sure, but I think this will only work with the current names for the couplers [0,1,3,4]